### ISSUE TITLE
Can I use codeT5 as a pretrained model to do my code search work

### ISSUE THREAD

**Issue by wangsiqidahaoren** [salesforce/CodeT5#127]

**Comment by yuewang-cuhk** [comment]
Yes, we have released the embedding model of CodeT5+ and the running scripts to evaluate it on code search tasks. See [here](https://github.com/salesforce/CodeT5/tree/main/CodeT5%2B#text-to-code-retrieval) for more details:)
